iambic/config/wizard.py

```diff
@@ -1853,8 +1853,9 @@ def configuration_wizard_change_detection_setup(self, aws_org: AWSOrganization):
             return
 
         session, _ = self.get_boto3_session_for_account(aws_org.org_account_id)
+        # TODO: cloudtrail is not crossregion, so we need to use us-east-1
```

Comment:
Delete comments?

Reply:
I think this comment could help to understand why we are fixing the region

Reply:
Cool, in that case minor nit would be to: 1. Remove "TODO", and 2. `s/crossregion/cross-region/`